GoldRush.js

```diff
@@ -0,0 +1,80 @@
+// const Matrix = require('./Matrix')
+
+class GoldRush extends Matrix {
+
+    constructor() {
+        super(10, 10, 'Coin')
```

Comment:
better to make it more generic and add a `const` for the size of the matrix (so you or maybe the player can change)